clients/notifications/client.go

```diff
@@ -64,6 +64,7 @@ type Notification struct {
 	Category    CategoryEnum `json:"category"`              // Allows for categorization of notifications
 	Severity    SeverityEnum `json:"severity"`              // Denotes the severity of a notification
 	Content     string       `json:"content"`               // Content contains the body of the notification
+	ContentType string       `json:"contentType,omitempty"` // ContentType contains the content type of the notification
```

Comment:
This need to be `json:"contenttype,omitempty"` to match the model:
https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/models/notifications.go#L34

Reply:
Oops.  Kinda surprised that worked.  Just force pushed.